examples/file_uploads/create_file_uploads_extenal.py

```diff
@@ -0,0 +1,93 @@
+import os
+import time
+
+from notion_client import Client
+
+# Initialize the client
+NOTION_TOKEN = os.getenv("NOTION_TOKEN", "")
+notion = Client(auth=NOTION_TOKEN)
+
+
+def _wait_for_upload_completion(
+    file_upload_id: str, poll_interval: int = 5, max_wait_time: int = 300
+):
+    """
+    Wait for file upload/import to complete.
+
+    Args:
+        file_upload_id: The file upload ID.
+        poll_interval: Polling interval in seconds.
+        max_wait_time: Maximum wait time in seconds.
+    """
+    start_time = time.time()
+
+    while time.time() - start_time < max_wait_time:
```

Comment:
I am not a maintainer of this project, just a user.

That said, I recommend you consider using [`time.monotonic`](https://docs.python.org/3/library/time.html#time.monotonic).

`time.time()` can jump forwards or go backwards.